template/.vscode/settings.json

```
@@ -0,0 +1,7 @@
+{
+    "rust-analyzer.rustfmt.overrideCommand": [
+        "rustfmt",
+        "+nightly-2025-01-15",
```

note: We should use the `rust_nightly_version` variable here.

```suggestion
        "{[rust_nightly_version}]",
```

This also means that this file needs to be renamed to `settings.json.j2`